server/routes/server.routes.js

```diff
@@ -156,4 +144,55 @@ router.get('/:username/collections', (req, res) => {
   ));
 });
 
+
+// Mobile Routes
+if (process.env.MOBILE_ENABLED) {
+  router.get('/mobile', (req, res) => res.send(renderIndex()));
```

Comment:
If these routes are the same as the desktop routes above, one way to avoid duplication would be to mount them again under the `/mobile` namespace in server.js:

```
// this is supposed to be TEMPORARY -- until i figure out
// isomorphic rendering
app.use('/', serverRoutes);

if (process.env.MOBILE_ENABLED) {
  app.use('/mobile’, serverRoutes);
}
```

Reply:
Wow this is amazing, I didn't know this was possible, thanks andrew!